### DISCUSSION TITLE
Video connecting the schema of OSPOS to MySQL Workbench?

### DISCUSSION THREAD

**Discussion by duchuynh** [opensourcepos/opensourcepos#3856] [General]
Hi, can someone possibly create a quick video on connecting the OSPOS database to MySQL workbench?

**Comment by jekkos** [comment]
I don't think this is different for any other mysql application, you can just search for generic instructions on the internet.

**Comment by duchuynh** [comment]
Hi @jekkos , do we use the database user, password, and hostname from the database.php file to connect to the database? My 

![image](https://github.com/opensourcepos/opensourcepos/assets/1304489/b6eff116-980b-4c1b-9d02-51d0ca3ae3e2)

I used the same username, password, connection name, but changed hostname to the ip address that our business hosts ospos. However, we still get an error.

**Reply by duchuynh** [reply]
We are hosting the OSPOS on a separate linux virtual machine, while we are trying to access the server on a workbench on a windows machine.

**Reply by jekkos** [reply]
that should work if you enable remote access for the use you have configured in mysql

**Reply by duchuynh** [reply]
Hi @jekkos Thanks for this. That was the problem I was having that I did not know about. So MySQL by default, does not accept access from other virtual machines and servers.

I used this video to change the configurations and was able to connect to the database from another virtual machine.

https://www.youtube.com/watch?v=PVo70d-BkUg
